pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -15,8 +15,6 @@ in
 
   agda2-mode = callPackage ./manual-packages/agda2-mode { };
 
-  beancount = callPackage ./manual-packages/beancount { };
```

Comment:
I cannot find `beancount` on GNU ELPA or GNU-devel ELPA.

Reply:
It is on NonGNU ELPA: https://elpa.nongnu.org/nongnu/beancount.html

https://github.com/NixOS/nixpkgs/pull/325055/files#diff-feb07e0cf8a541b8fcc5b0a76d87bd9d37cdc81709e778f9de14370f4be69542R215